packages/webdriverio/src/utils.js

```diff
@@ -166,10 +166,17 @@ export const getPrototype = (scope) => {
 
 /**
  * get element id from WebDriver response
- * @param  {object} res         body object from response
- * @return {string|undefined}   element id or null if element couldn't be found
+ * @param  {?Object|undefined} res         body object from response or null
+ * @return {?string}   element id or null if element couldn't be found
  */
 export const getElementFromResponse = (res) => {
+    /**
+    * a function selector can return null
+    */
+    if (!res) {
```

Comment:
@jrobinson01 @christian-bromann can **res** be boolean?
I mean if res === false we will return null.

Reply:
It shouldn’t .. it is usually the request response body